airbyte-cdk/java/airbyte-cdk/README.md

```diff
@@ -145,6 +145,7 @@ Maven and Gradle will automatically reference the correct (pinned) version of th
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                                        |
 |:--------|:-----------|:-----------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0.26.1  | 2024-03-20 | [\#36340](https://github.com/airbytehq/airbyte/pull/36340) | Change in CdcMetadataInjector interface.                                                                                                                       |
```

Comment:
In a changelog, it's useful to describe whether the change is functional or not. In this case, it's purely non-functional.